modules/distr-tracing-otel-troubleshoot-logging-exporter.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-troubleshooting.adoc
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-troubleshoot-logging-exporter_{context}"]
+= Logging exporter
+
+You can configure the logging exporter to export the collected data to the standard output.
+
+.Procedure
+
+. Configure the OpenTelemetry Collector custom resource as follows:
++
+[source,yaml]
+----
+  config: |
+    exporters:
+      logging:
+        verbosity: detailed
+    service:
+      pipelines:
+        traces:
+          exporters: [logging]
+        metrics:
+          exporters: [logging]
+        logs:
+          exporters: [logging]
+----
+
+. Use the `oc logs` command or the OpenShift console to export the logs to the standard output.
```

Comment:
Nit: I think it would benefit customers to include substeps that show the `oc logs` command in a command block and another substep to show how to access logs in the console.